chennai.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+header: Reaching the Venues
+title: Reaching the Venues
+bloglike: true
+---
+
+This page is to help the attendees reach Chennai and the conference venues.
+
+<br>
+
+### Getting to PyCon India 2019
+
+**By Plane**
+
+Chennai has an international airport that is well connected to Europe, the Middle East, US,
+and South East Asia. It is also well connected by domestic flights.
+
+**Chennai Trade Center(Conference)**
+
+The easiest option is to take a metro from Chennai Airport towards Washermenpet Metro and alight
+at Guindy Metro. From the metro station, you can board one of these buses - 49, 54, 54D and
+get down at Contonment Cemetery. The venue is 200 meters from here. Alternatively, you can hail
+a cab from ride sharing services like Uber or Ola. Please **do not** try to get an autorickshaw
+as they usually rip off outsiders. Same goes for rental cabs at the airport as well.
+
+**IITM Research Park(Workshops and Dev Sprints)**
+
+Again, the easiest option is to take a metro towards Washermenpet and alight at Saidapet Metro.
+From there, a bus towards Besant Nagar should take you to IIT Madras. The bus routes are
+5E, 23C and 47. The IIT-M bus stop might also be referred to CLRI.
+
+<br>
+
+---
+
+<br>
+
+#### By Train
+
+Chennai's Central Railway Station is where you will most probably arrive. 
+
+**Chennai Trade Center(Conference)**
+
+The simplest way is to take a bus towards Poonamallee(bus route 54) and alight at Contonment
+Cemetery(or ask for Chennai Trade Center itself). It's an hour's ride and the venue is about 200
+meters from the bus stop.
+
+**IITM Research Park(Workshops and Dev Sprints)**
+
+From Chennai Central, you can take a metro ride to Saidapet Metro and catch a bus
+towards Besant Nagar and alight at IIT-M or CLRI. The bus routes are 19B, 5C, 5E, 23C and 47.
```

Comment:
MRTS is another option. The closes MRTS station is Thiruvanmiyur station.

Reply:
Indiranagar MRTS stop which is one stop before Thiruvanmiyur also closest one for RAmanujam IT park, we can give that option also. Ramnujam IT park is almost at the center between those two stations